tictactoe.html

```diff
@@ -0,0 +1,53 @@
+<!--#
+    set var="title"     value="Tic Tac Toe"             --><!--#
+    set var="header"    value="Tic Tac Toe"             --><!--#
+    set var="subheader" value="Have fun!"               --><!--#
+    set var="tictactoe" value="active"                  --><!--#
+    set var="link"      value="tictactoe/tictactoe.css" --><!--#
+
+    include virtual="ssi/header.html" --><!--#
+    include virtual="ssi/nav.html"    -->
+
+<article>
```

Comment:
I recommend using an `id` here to label the entire TTT component. Then switch `id` to `class` below within the HTML of this component.